src/Components/SmartComponents/CVEs/CVEs.test.js

```diff
@@ -86,14 +86,14 @@ const customMiddleWare = store => next => action => {
 
 const mockStore = configureStore([customMiddleWare]);
 const store = mockStore(initialState);
-
+jest.spyOn(console, 'warn').mockImplementation(() => {});
+jest.spyOn(console, 'error').mockImplementation(() => {});
```

Comment:
Does this reduce console spam during test runs? Would it make sense to implement these in more tests?

Reply:
Yup, this does help to hide those errors. I had to add this as tests were bombing my laptop. Lesser logs will help to ease the process. However, there is a trade-off, we will not be able to see warnings and errors above the tests summary table. All the errors are not affected below that table.